temba/contacts/migrations/0028_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contacts', '0022_auto_20151116_1433'),
+        ('contacts', '0027_auto_20151103_1014'),
+    ]
+
+    operations = [
+    ]
```

Comment:
We don't want a merged migration histories, instead rename your migrations (and their number) to occur at the right points in the migration history.